dask/bytes/tests/test_s3.py

```diff
@@ -193,32 +201,32 @@ def test_getsize(fmt):
 
 def test_modification_time_read_bytes():
     with s3_context('compress', files) as s3:
-        _, a = read_bytes('compress/test/accounts.*', s3=s3)
-        _, b = read_bytes('compress/test/accounts.*', s3=s3)
+        _, a = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
+        _, b = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
```

Comment:
Hrm, ok, how do you suggest that we resolve this difference in opinion?

My argument for is that it seems to be common practice to include many keyword arguments in the main function as long as it's clear to where they need to be forwarded.  

In practice I find writing `storage_options={'s3': s3}` to be verbose.  While I agree that it is more explicit it also feels awkward.  Taking this to a further extreme we could separate all options into two dicts to be completely explicit

    _, c = read_bytes(parsing_options={...}, storage_options={...})

This is obviously an overly-extreme example but I hope that it illustrates my point.